scripts/prompt_injection/setup.py

```
@@ -0,0 +1,41 @@
+    },
+    entry_points={
+        "console_scripts": [
+            "prompt-detector=prompt_injection.detector:main",
+            "prompt-api=prompt_injection.api:main",
+            "prompt-guard=prompt_injection.llm_guard:main",
+        ],
+    },
+    author="OpenHands",
+    author_email="openhands@all-hands.dev",
+    description="Tools for detecting and mitigating prompt injection attacks",
+    long_description=open("README.md").read(),
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)